src/main.ts

```diff
@@ -17,18 +17,33 @@ async function run(): Promise<void> {
     const octokit = github.getOctokit(inputs.token);
 
     const ownership = {
-      owner: github.context.repo.owner,
-      repo: github.context.repo.repo,
+      owner: '',
+      repo: '',
     };
-    const sha = github.context.sha;
+    let sha = '';
+
+    if (inputs.repo) {
+      const repo = inputs.repo.split('/');
```

Comment:
Would be nice to be sure we can actually split it, would you mind adding a check in `src/inputs.ts`?

Reply:
And fixed, also added a test for this.